pages/about.html

```diff
@@ -3,10 +3,31 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link href="styles/style.css" rel="stylesheet">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>about</title>
 </head>
 <body>
-  
+  <div id="navbar">
+    <h1>Yang Su</h1>
+    <ul>
+      <li class="nav-font"><a href="portfolio.html">Portfolio</a></li>
+      <li class="nav-font"><a href="index.html">Home</a></li>
+    </ul>
+  </div>
+  <div id="dec">
+    <section>
+      <h1 style="color:cadetblue; text-align: center;">Connect</h1>
+      <p id="message">Emailing me is the best way to ensure I’ll respond eventually!</p>
+      <p><img class="connect" src="content/Octocat.png"><a class="connect" href="https://github.com/guaiguaiyang?tab=repositories">Github</a></p>
+      <p><img class="connect" src="content/linkdin.png"><a class="connect" href="https://www.linkedin.com/in/yang-su-9b7527179/">Linkedin</a></p>
+      <p><img class="connect" src="content/mailbox.png"><a class="connect" href="mailto:suyang.sophia@outlook.com" role="link">Email</a></p>
```

Comment:
All `img` tags should have alt text. This is used by people who are blind or have low vision who depend on screen readers. The screen reader will read the alt text to explain the picture to the user. If there is no alt text, that person will not know what the picture is.